src/_nebari/stages/infrastructure/template/gcp/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "4.83.0"
+      version = "4.8.0"
```

Comment:
Any specific reason for this downgrade? 

This is 2 years old: https://registry.terraform.io/providers/hashicorp/google/4.8.0/docs

Reply:
> I think we can circle back to this issue 

Yep, sure. Let's not block this release.